### DISCUSSION TITLE
Whether to add some support on other Operating Systems like macOS ?

### DISCUSSION THREAD

**Discussion by ghost** [Tualin14/baPlayer#32] [Development]
**Description**
When I was using my old Windows PC, the program works well.But when I want to use it on my Mac, it doesn't work at all(I can't run all the programs that end with ".exe" because of the macOS).

So how can I run Aris Studio on my Mac without vitural machines?

My English is not good, so don't laugh at me. :P

Thank you.

**Comment by ttttttttttttttllllllllllllll** [comment]
生成mac程序需要苹果电脑，但是我没有，所以生不成。目前我已知的是 [豪斯·比格朗费斯的个人空间-哔哩哔哩](https://b23.tv/JBUDMKR) 用的mac，具体操作方法你问问他。这个需要下载unity和克隆代码，来在mac的unity运行。

**Reply by ghost** [reply]
好的，谢谢您。
我去问问这位sensei。

**Comment by kiraio-moe** [comment]
Technically, we can support it. But we don't have any Mac device to test Aris Studio if it's work well or not.
If we're forced to do so, it's like walking blindly in a void space...
